### DISCUSSION TITLE
How to activate license purchased

### DISCUSSION THREAD

**Discussion by Halcyon-855** [rustdesk/rustdesk#8073] [General]
Hello, I have just purchased a Rustdesk license to get a faster connection speed and better service. But I do not find some place to use license, neither in the installed Rustdesk on my Windows laptop, nor in the web console. So I wonder how to set up my Rustdesk with my purchased license.
By the way, I need to use my Windows laptop to control a ubuntu PC.
Thanks

**Comment by rustdesk** [comment]
![Screenshot_20240517_124808](https://github.com/rustdesk/rustdesk/assets/71636191/680bbed4-a576-4aaf-989c-8b69cdf9ab9b)
